orchagent/orchdaemon.cpp

```diff
@@ -654,3 +663,50 @@ bool OrchDaemon::warmRestartCheck()
     gSwitchOrch->restartCheckReply(op,  data, values);
     return ret;
 }
+
+void
+OrchDaemon::check_and_exit()
+{
+    SWSS_LOG_ENTER();
+
+    swss::SubscriberStateTable tbl(m_configDb, "CONTAINER_FEATURE");
+    std::deque<KeyOpFieldsValuesTuple> entries;
+
+    tbl.pops(entries);
```

Comment:
You may use simply Table if you don't subscribe.

Reply:
Thanks. This helped cut down lines of code. 
BTW, I have not tested my changes yet. After it passes review, I will do one more final test.